quodlibet/quodlibet/ext/events/mpdserver/main.py

```diff
@@ -310,6 +310,10 @@ def status(self):
             state = "stop"
 
         status = [
+            ("audio", "%d:%d bps:%d" % (
```

Comment:
MPD seems to just do "%d:%d:%d": https://github.com/MusicPlayerDaemon/MPD/blob/44c60567dd7957b440c7643797bc223cf5aceb6e/src/AudioFormat.cxx#L59

Any reason for adding the "bps" bit?

Reply:
I can drop bps bit.